src/test/java/org/jenkinsci/plugins/workflow/job/WorkflowJobTest.java

```diff
@@ -93,9 +94,17 @@ public void addAction() throws Exception {
         assertTrue(p.isDisabled());
         assertFalse(p.isBuildable());
         HtmlForm form = wc.getPage(p, "configure").getFormByName("config");
-        HtmlCheckBoxInput checkbox = form.getInputByName("disable");
-        assertTrue(checkbox.isChecked());
-        checkbox.setChecked(false);
+        DomNode domNode = form.querySelector("input[name='enable']");
+        if (domNode != null) {
+            HtmlCheckBoxInput checkbox = (HtmlCheckBoxInput) domNode;
+            assertFalse(checkbox.isChecked());
+            checkbox.setChecked(true);
+        } else {
+            // TODO remove compatibility when baseline includes https://github.com/jenkinsci/jenkins/pull/6485
```

Comment:
Also here—not clear to a reader without digging into the upstream PR which version is old and which is new.

Reply:
> not clear to a reader without digging into the upstream PR which version is old and which is new.

You mean a TODO comment lacked sufficient context for someone not already familiar with the problem? Yes, just as in https://github.com/jenkinsci/jenkins-test-harness/pull/428#discussion_r863019143. In that PR, you did not respond to the review feedback.